about/overview-openshift-serverless-logic.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="overview-openshift-serverless-logic"]
+= Openshift Serverless Logic overview
+:context: overview-openshift-serverless-logic
+
+
+toc::[]
+
+OpenShift Serverless Logic enables developers or architects to define declarative workflow models that orchestrate event-driven, serverless applications.
+
+OpenShift Serverless Logic is also designed to write workflows in formats (JSON or YAML) that might be better suited for developing and deploying serverless applications in cloud or container environments.
+
+For OpenShift Serverless Logic implementation, you can use the Kubernetes operator to deploy the workflow models using Knative. Knative Eventing plays an important role and provides the essential infrastructure for event-driven architectures.
```

Comment:
```suggestion
OpenShift Serverless Logic provides the Kubernetes operator to deploy & manage the workflow models using Knative. Knative Eventing plays an important role and provides the essential infrastructure for event-driven architectures.
```
Small suggestion, let me know.
We should also mentions serving somehow imho.
Lastly, from user perspective it did not tell me much, if we are talking overview I think it could be expanded slightly.

Reply:
Maybe we can say something like this:


OpenShift Serverless Logic provides the Kubernetes operator to deploy & manage the workflow models using Knative Serving or Kubernetes deployments.  And, Knative Eventing provides the essential infrastructure for event-driven architectures.